samples/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
+    <SignAssembly>false</SignAssembly>
```

Comment:
Note: MAUI currently requires this because the MAUI dependencies are unsigned.  But it's probably good form to not be signing our sample apps anyway.